dominio/dominio/SolucionSOLID/Alumno/AlumnoCreation.java

```diff
@@ -0,0 +1,62 @@
+package dominio.SolucionSOLID.Alumno;
+
+import dominio.Alumno;
+import dominio.SolucionSOLID.GestorInstituto;
+import exceptions.*;
+
+public class AlumnoCreation implements IAlumnoCreation {
+    private AlumnoValidationExisting alumnoValidador;
+    //private Alumno alumnoLocal;
+
+    //Aca se inyect dependencia con la Class ALumnovalidador
+    public AlumnoCreation(AlumnoValidationExisting alumnoValidador) {
+        this.alumnoValidador = alumnoValidador;
+    }
+
+    //check si realmente es necesario this method
+    @Override
+    public boolean addAlumno(String nombre, String apellido, String dni, String telefono, GestorInstituto gestor) {
+
+        try {
+            Alumno alumno = new Alumno(nombre, apellido, dni, telefono);
+
+            alumnoValidador.validar(alumno, gestor);
+            return gestor.listaAlumnos.add(alumno);
+
+        }catch (DniInvalidoException e2) {
+            return false;
+        }catch (NombreVacioException e2) {
+            return false;
+        }catch (ApellidoVacioException e2) {
+            return false;
+        }catch (TelefonoInvalidoException e2) {
+            return false;
+        }catch (PrincipalException e) {
+            return false;
```

Comment:
En esta serie de Catch, siendo que el valor devuelto es el mismo, no tiene sentido diferenciar. Con Capturar solo "PrincipalException" se contemplan todos los casos
```suggestion
        }catch (PrincipalException e) {
            return false;
```